components/eam/bld/configure

```diff
@@ -1737,7 +1737,8 @@ if (defined $opts{'use_MMF'}) {
     my $crm_adv = $cfg_ref->get('crm_adv');
     my $yes3Dval = 1;
     if ($crm_ny eq 1) {$yes3Dval = 0;}
-    $cfg_cppdefs .= " -D_$crm_adv -D$MMF_microphysics_scheme -DYES3DVAL=$yes3Dval";
+    if (defined $opts{'crm_adv'}) { $cfg_cppdefs .= " -D_$crm_adv "; }
```

Comment:
is ';' before '}' a typo?

Reply:
actually the last statement in the { ... } block is ok to go without a semicolon. But no harm to have it.